src/main/java/slashcommands/SlashCommandRegister.java

```diff
@@ -1,11 +1,12 @@
 package slashcommands;
 
+import language.LanguageCommand;
+import language.LanguageLogic;
 import logic.*;
 import pw.mihou.velen.interfaces.Velen;
 
 public class SlashCommandRegister {
-
-    public static Velen setupVelen() {
+    public static Velen setupVelen(LanguageLogic languageLogic) {
```

Comment:
Why is the setup function for all slash commands passing in a language logic?

Reply:
Because the autocomplete will be outside of the setup Velen bit, and it's an instance.